projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -26,14 +26,33 @@ const GlobalNotice = ( { message, options, clean } ) => {
 	 * `success`, `info`, `warning`, `error`
 	 */
 	const iconMap = {
-		error: warning,
+		error: (
+			<svg
```

Comment:
Same point about using `SVG` and `Path`.

Reply:
Addressed in 1874cda